Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,377 @@
+  parameter Modelica.SIunits.MassFlowRate mCW_flow_nominal
+    "Nominal condenser water mass flow rate"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.Pressure dpCW_nominal
+    "Pressure difference at the condenser water side"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.Temperature TAirInWB_nominal
+    "Nominal air wetbulb temperature"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.Temperature TCW_nominal
+    "Nominal condenser water temperature at tower inlet"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.TemperatureDifference dT_nominal
+    "Temperature difference between inlet and outlet of the tower"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.TemperatureDifference dTApp
+    "Approach temperature"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.Temperature TMin
+    "Minimum allowed water temperature entering chiller"
+    annotation (Dialog(group="Cooling Tower"));
+  parameter Modelica.SIunits.Power PFan_nominal
+    "Fan power"
+    annotation (Dialog(group="Cooling Tower"));
+  // pump parameters
+  replaceable parameter Buildings.Fluid.Movers.Data.Generic perCHWPum
+    constrainedby Buildings.Fluid.Movers.Data.Generic
+    "Performance data of chilled water pump"
+    annotation (Dialog(group="Pump"),choicesAllMatching=true,Placement(transformation(extent={{164,264},{178,278}})));
+  replaceable parameter Buildings.Fluid.Movers.Data.Generic perCWPum
+    constrainedby Buildings.Fluid.Movers.Data.Generic
+    "Performance data of condenser water pump"
+    annotation (Dialog(group="Pump"),choicesAllMatching=true,Placement(transformation(extent={{204,264},{218,278}})));
+  parameter Modelica.SIunits.Pressure dpCHWPumVal_nominal
+    "Nominal pressure drop of chilled water pump valve"
+    annotation (Dialog(group="Pump"));
+  parameter Modelica.SIunits.Pressure dpCWPumVal_nominal
+    "Nominal pressure drop of condenser water pump valve"
+    annotation (Dialog(group="Pump"));
+  // control settings
+  parameter Modelica.SIunits.Time tWai
+    "Waiting time"
+    annotation (Dialog(group="Control Settings"));
+  parameter Modelica.SIunits.PressureDifference dpSetPoi(
+    displayUnit="Pa")
+    "Demand side pressure difference setpoint"
+    annotation (Dialog(group="Control Settings"));
+  // dynamics
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.DynamicFreeInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state"
+    annotation (Evaluate=true,Dialog(tab="Dynamics",group="Equations"));
+  parameter Modelica.Fluid.Types.Dynamics massDynamics=energyDynamics
+    "Type of mass balance: dynamic (3 initialization options) or steady state"
+    annotation (Evaluate=true,Dialog(tab="Dynamics",group="Equations"));
+  Medium.ThermodynamicState sta_a(
```

Comment:
@JingWang-CUB please see the 2nd round of errors I added to [Issue Comment 844309702](https://github.com/khinkelman/modelica-buildings/pull/34#issuecomment-844309702) above. It appears accessing the conditional ports is not correct Modelica. I wonder if these conditional `ThermodynamicStates` would need to be added to the base class `PartialPlant` instead? 

My suggestion: Delete lines 89-108 from this `Plant.mo` model. We can notify Antoine of this issue and get his input whether he wants `show_T` capabilities at the plant ports while compiling models with JModelica.

What do you think? Thanks!

Reply:
Looks like there's no way for us to access these conditional ports here. We can remove lines 89-108  together with 10-12 since we already have the temperature sensors for supply and return. Notifying Antoine of this issue sounds good to me. Thanks!
@ChengnanShi-Work could you please delete the lines?